pybench/pybench.py

```diff
@@ -12,11 +12,17 @@
 import time
 from twisted.internet import (defer,
                               reactor)
+from twisted.internet.ssl import ClientContextFactory
 from twisted.web.client import Agent
 from twisted.web.http_headers import Headers
 from urlparse import urlparse
 
 
+class WebClientContextFactory(ClientContextFactory):
+    def getContext(self, hostname, port):
+        return ClientContextFactory.getContext(self)
```

Comment:
good call. I added some docstrings to clarify. The class isn't strictly necessary for our purposes, but it basically disables the ssl certificate checking so that this script can be used in dev environments without signed certs.